articles/manual/statements.md

```diff
@@ -672,3 +760,130 @@ X(q);
 CNOT(control, target);
 Adjoint T(q);
 ```
+## Measurements ##
+
+Using the `Measure` operation, which is a built in intrinsic non-unitary operation, we can extract classical information from an object of type `Qubit` and assign a classical value as a result, which has a reserved type `Result`, indicating that the result is no longer a quantum state. 
+The input to `Measure` is a Pauli axis on the Bloch sphere, represented by an object of type `Pauli` (i.e., for instance `PauliX`) and an object of type `Qubit`. 
+
+A simple example is the following operation which creates one qubit in the $\ket{0}$ state, then applies a Hadamard gate ``H`` to it and then measures the result in the `PauliZ` basis. 
+
+```qsharp
+operation MeasurementOneQubit () : Result {
+
+    // The following using block creates a fresh qubit and initializes it 
+    // in the |0〉 state.
+    using (qubit = Qubit()) {
+        // We apply a Hadamard operation H to the state, thereby creating the 
+        // state 1/sqrt(2)(|0〉+|1〉). 
+        H(qubit); 
+        // Now we measure the qubit in Z-basis.
+        let result = M(qubit);
+        // As the qubit is now in an eigenstate of the measurement operator, 
+        // we reset the qubit before releasing it. 
+        if (result == One) { X(qubit); }   
+        // Finally, we return the result of the measurement. 
+        return result;
+    }
+}
+```
+
+A slightly more complicated example is given by the following operation which returns the Boolean value `true` if all qubits in a register of type `Qubit[]` are in the state zero, when measured in a specified Pauli basis and `false` otherwise. 
```

Comment:
```suggestion
A slightly more complicated example is given by the following operation which returns the Boolean value `true` if all qubits in a register of type `Qubit[]` are in the state zero when measured in a specified Pauli basis, and `false` otherwise. 
```